src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/StandardRequestManager.java

```diff
@@ -62,50 +67,25 @@
 
     private IToken<?> requestableTypeRequestResolverAssignmentDataStoreId;
 
+    private IToken<?> playerRequestResolverId;
+
+    private IToken<?> retryingRequestResolverId;
+
     private IDataStoreManager dataStoreManager;
 
     /**
      * Colony of the manager.
      */
     @NotNull
     private final IColony colony;
-    /**
-     * The fallback resolver used to resolve directly to the player.
-     */
-    @NotNull
-    private IPlayerRequestResolver                       playerResolver          = null;
-    /**
-     * The fallback resolver used to resolve using retries.
-     * Not all requests might support this feature, requests that do should implement {@link IRetryable} on their requestable.
-     * Anything that implements {@link IDeliverable} is by definition retryable.
-     */
-    @NotNull
-    private IRetryingRequestResolver retryingResolver = null;
 
     @NotNull
     private int version = -1;
 
     public StandardRequestManager(final IColony colony)
     {
         this.colony = colony;
-        setup();
-
-        this.playerResolver = getFactoryController().getNewInstance(TypeConstants.PLAYER_REQUEST_RESOLVER, this);
-        this.retryingResolver = getFactoryController().getNewInstance(TypeConstants.RETRYING_REQUEST_RESOLVER, this);
-        ResolverHandler.registerResolver(this, this.playerResolver);
-        ResolverHandler.registerResolver(this, this.retryingResolver);
-    }
-
-    /**
-     * Constructor for unit tests.
-     */
-    StandardRequestManager()
-    {
-        this.colony = null;
-        this.playerResolver = null;
-        this.retryingResolver = null;
-
-        setup();
+        reset();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this call from a constructor to the overridable "reset" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1699)